A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -393,9 +395,12 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
             this._onModeManagedSpeed();
         }
         if (_event === "MODE_SELECTED_HEADING") {
-            if (this.flightPlanManager.getWaypointsCount() === 0) {
-                return;
-            }
+            SimVar.SetSimVarValue("L:A32NX_GOAROUND_HDG_MODE", "bool", 1);
+            SimVar.SetSimVarValue("L:A32NX_GOAROUND_NAV_MODE", "bool", 0);
+            //why is below code checking for waypointcounts when we are in selected mode?
+            //if (this.flightPlanManager.getWaypointsCount() === 0) {
+            //    return;
+            //}
```

Comment:
Remove commented code

Reply:
this i would like to keep together with the comment ... i still dont understand why asobo had this code there in the first place ... maybe i miss something now and i need to comment it in again? If we all agree its nonsense, i delete it though ...